docs/pages/repo/docs/handbook/dev.mdx

```diff
@@ -112,6 +112,36 @@ For example:
 turbo run dev --filter docs
 ```
 
+## Running setup tasks
+
+There may be some tasks that you need to run _before_ your persistent, long-running development tasks are running. Some examples of setup steps would be:
```

Comment:
A few distinctions here that I think should be clarified

```suggestion
There may be some tasks that you need to run across your monorepo _before_ your persistent, long-running development tasks are started. Some examples of setup steps would be:
```

Also should we call this `prepare-dev` or `dev-setup`? `pre` (and `post`) are pretty similar to package manager script prefix hooks that could make your life really confusing if you omit the dash.

Reply:
Oh, yeah, good call.